### DISCUSSION TITLE
WebDAV support?

### DISCUSSION THREAD

**Discussion by TRRAXT** [veeso/termscp#229] [Ideas]
Yeah, that's a bit weird, but perhaps you could consider adding WebDAV support?
Your terminal so awsome that I can't use anything else xD

**Comment by veeso** [comment]
It could be implemented

https://github.com/veeso/termscp/issues/231

**Comment by veeso** [comment]
Implemented in 0.13.0 which will soon be available

**Reply by TRRAXT** [reply]
Thousand thanks!
